test/cluster-store.js

```diff
@@ -2,7 +2,7 @@ var cluster = require('cluster');
 var expect = require('chai').expect;
 var ioServer = require('socket.io');
 var ioClient = require('socket.io-client');
-var debug = require('debug')('test');
+var debug = require('debug')('strong-cluster-socket.io-store:test');
```

Comment:
As you wish. The scheme above is consistent with the rest of the strong-* repos, fwiw. I'm surprised you would run unit tests with DEBUG output, but no want the actual output from the unit tests, I usually do the opposite (want the top-level output from the test code, without the noise of all the underlying libraries, `DEBUG=*:test`). 

`DEBUG=*,-*:test` might work for the opposite.

But its no big deal either way.